modules/images-create-s2i-build.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
-//* assembly/openshift_images
-//* assembly/builds/build-strategies.adoc
+//
+//* builds/build-strategies.adoc
 
 [id="images-create-s2i-build_{context}"]
 = Understanding the s2i build process
```

Comment:
S2I? The previous section styled them all as "S2I," and the headings in the next section do not. I'm not going to comment on all of them, but if you're up for changing titles, too, please update them.